docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -32,59 +32,65 @@ Feathr will get the configurations in the following order:
+| REDIS_PASSWORD                                                          | Password for the Redis cluster.                                                                                                                                                                                                                            | Required if using Redis as online store.                                                                                |
+| FEATURE_REGISTRY__API_ENDPOINT                                          | Specifies registry endpoint.                                                                                                                                                                                                                               | Required if using registry service.                                                                                     |
+| FEATURE_REGISTRY__PURVIEW__PURVIEW_NAME  (Deprecated Soon)              | Configure the name of the purview endpoint.                                                                                                                                                                                                                | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details. |
+| FEATURE_REGISTRY__PURVIEW__DELIMITER  (Deprecated Soon)                 | See [here](#FEATURE_REGISTRY__PURVIEW__DELIMITER) for more details.                                                                                                                                                                                        | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details. |
+| FEATURE_REGISTRY__PURVIEW__TYPE_SYSTEM_INITIALIZATION (Deprecated Soon) | Controls whether the type system (think this as the "schema" for the registry) will be initialized or not. Usually this is only required to be set to `True` to initialize schema, and then you can set it to `False` to shorten the initialization time.  | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details. |
+| FEATHR_MAVEN_VERSION                                                    | Version number like `0.9.0`. Used to define maven package version when main jar is not defined.                                                                                                                                                            | Optional if custom maven package version is needed.                                                                     |
```

Comment:
Suggest change `Optional if custom maven package version is needed.` to just Optional